components/sidebar/sidebar_whats_new_modal/sidebar_whats_new_modal.scss

```diff
@@ -1,12 +1,6 @@
 @charset 'UTF-8';
 
 .GenericModal {
```

Comment:
All this .GenericModal shouldn't be here unless it is inside something like `.SidebarWhatsNewModal` class or something like that (not related to your changes, but something that surprise me).

Reply:
@devinbinnie you added this changes. This are changing the `.GenericModal` and the `.modal` classes globally from this place, I think this shouldn't happen. I would expect to have a parent class `.SidebarWhatsNewModal` or a `.SidebarWhatsNewModal.GenericModal` and `.SidebarWhatsNewModal.modal`.

Reply:
You're correct. I think it only works because we never have both components loaded at the same time. But yes the modal should be using something like `.GenericModal.SidebarWhatsNewModal`